test/cmd/create.sh

```diff
@@ -134,8 +134,9 @@ run_kubectl_create_kustomization_directory_tests() {
   set -o errexit
 
   ## kubectl create -k <dir> for kustomization directory
-  # Pre-condition: no ConfigMap, Deployment, Service exist
-  kube::test::get_object_assert configmaps "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
scope to the `test-the-map` configmap with a field-selector

Reply:
done